### ISSUE TITLE
1.0.8 breaks flag usage on onSuccess

### ISSUE THREAD

**Issue by nanoxd** [gilamran/tsc-watch#6]
Previously with v1.0.7, the following command would work:

`tsc-watch --onSuccess \"node -r dotenv/config dist/index.js\"`

As of 1.0.8 it fails with the following error message:
```
yarn dev v0.27.5
$ tsc-watch --onSuccess "node -r dotenv/config dist/index.js"
env: node\r: No such file or directory
error Command failed with exit code 127.
```

**Comment by fabiob** [comment]
Duplicate of #5

**Comment by gilamran** [comment]
As @fabiob mentioned it's Duplicate of #5 (Fixed in version 1.0.10)
